guides/v2.1/release-notes/ReleaseNotes2.1.16CE.md

```diff
@@ -0,0 +1,212 @@
+
+<!--- ENGCOM-2928 -->*  `\Magento\Sales\Model\Validator` is now covered by unit tests. *Fix submitted by [Dmytro Cheshun](https://github.com/dmytro-ch) in pull request [17940](https://github.com/magento/magento2/pull/17940)*.
+
+
+
+
+### Theme
+
+<!--- ENGCOM-2917 -->* Magento now displays the wishlist icon on the shopping cart in mobile view. *Fix submitted by [Hitesh](https://github.com/hitesh-wagento) in pull request [17912](https://github.com/magento/magento2/pull/17912)*. [GitHub-17851](https://github.com/magento/magento2/issues/17851)
+
+
+
+<!--- ENGCOM-2325 -->* Multishipping checkout now works as expected. Previously, Magento displayed the `Shipping address is not set` error message  when checking out an order with multiple addresses. *Fix submitted by [Dmytro Cheshun](https://github.com/dmytro-ch) in pull request [16783](https://github.com/magento/magento2/pull/16783)*. [GitHub-16555](https://github.com/magento/magento2/issues/16555)
+
+
+### UI
+
+<!--- ENGCOM-2825 -->* The JavaScript validation rule used to validate AM/PM time settings now works as expected when JavaScript is minified. *Fix submitted by [Dmytro Cheshun](https://github.com/dmytro-ch) in pull request [17689](https://github.com/magento/magento2/pull/17689)*. [GitHub-17648](https://github.com/magento/magento2/issues/17648)
+
+<!--- ENGCOM-2835 -->* The message list component message type now has a message type of success. Previously, this type was always error when the parameters property was specified. *Fix submitted by [Dmytro Cheshun](https://github.com/dmytro-ch) in pull request [17702](https://github.com/magento/magento2/pull/17702)*. [GitHub-17700](https://github.com/magento/magento2/issues/17700)
+
+<!--- ENGCOM-2898 -->* The WYSIWYG editor now preserves the transparent background of .png  thumbnails. Previously, Magento rendered these transparent backgrounds black. *Fix submitted by [Eduard Chitoraga](https://github.com/eduard13) in pull request [17855](https://github.com/magento/magento2/pull/17855)*. [GitHub-14248](https://github.com/magento/magento2/issues/14248)
+
+
+
+## Community contributions
+
+We are grateful to the wider Magento community and would like to acknowledge their contributions to this release.
+
+### Individual contributor contributions
+
+The following table identifies contributions from our community members. This table lists the community member who contributed the pull request, the external pull request, and the GitHub issue number associated with the pull request (if available).
+
+{% include release-notes/engcom-2-1-16.md %}
+
+
+### Partner contributions
+
+The following table highlights contributions made by Partners. This table lists the Partner who contributed the pull request, the external pull request, and the GitHub issue number associated with it (if available).
+
+
```

Comment:
There's no table here. Delete this section?

Reply:
Getting this file to you ASAP for the include